examples/hetzner-machinedeployment.yaml

```diff
@@ -47,6 +47,9 @@ spec:
             # Optional
             datacenter: ""
             location: "fsn1"
+            # Optional: network IDs or names
+            networks:
+              - ""
```

Comment:
can we put something like `<<my-network>>` between the braces?

Reply:
Added `<< YOUR_NETWORK >>`.